index.yaml

```diff
@@ -2,262 +2,216 @@ indexes:
 
 # AUTOGENERATED
 
-# This index.yaml is automatically updated whenever the dev_appserver
-# detects that a new type of query is run.  If you want to manage the
-# index.yaml file manually, remove the above marker line (the line
-# saying "# AUTOGENERATED").  If you want to manage some indexes
-# manually, move them above the marker line.  The index.yaml file is
-# automatically uploaded to the admin console when you next deploy
-# your application using appcfg.py.
+# This index.yaml is automatically updated whenever the Cloud Datastore
```

Comment:
@kevjumba Per our discussion, confirming that the changes to this file look OK. They just seem to have been resorted and reformatted a bit, but the meaning should still be the same.

Reply:
Cool. Thanks!